### ISSUE TITLE
warning when using date or dateTime validation with moment.js

### ISSUE THREAD

**Issue by cateland** [ansman/validate.js#54]
Hello,

Using validate.js together with moment.js, i witnessed the following warning:
```
Deprecation warning: moment construction falls back to js Date. This is discouraged and will be removed in upcoming major release. Please refer to https://github.com/moment/moment/issues/1407 for more info.
```
here is a jsbin : http://jsbin.com/hobefijaza/2/edit?html,js,output

**Comment by ansman** [comment]
This warning is unavoidable, non valid dates will always produce this warning. There is a setting in moment.js that will disable this warning but there is nothing validate.js can do about it.
